src/openMVG/localization/CCTagLocalizer.cpp

```diff
@@ -247,6 +248,145 @@ bool CCTagLocalizer::localize(const image::Image<unsigned char> & imageGrey,
                   imagePath);
 }
 
+bool CCTagLocalizer::localizeAllAssociations(const std::unique_ptr<features::Regions> &genQueryRegions,
+                                              const std::pair<std::size_t, std::size_t> &imageSize,
+                                              const LocalizerParameters *parameters,
+                                              bool useInputIntrinsics,
+                                              cameras::Pinhole_Intrinsic_Radial_K3 &queryIntrinsics,
+                                              LocalizationResult & localizationResult,
+                                              const std::string& imagePath)
+{
+  namespace bfs = boost::filesystem;
+  
+  const CCTagLocalizer::Parameters *param = static_cast<const CCTagLocalizer::Parameters *>(parameters);
+  if(!param)
+  {
+    throw std::invalid_argument("The CCTag localizer parameters are not in the right format.");
+  }
+  
+  // it automatically throws an exception if the cast does not work
+  features::CCTAG_Regions &queryRegions = *dynamic_cast<features::CCTAG_Regions*> (genQueryRegions.get());
+  
+  std::map< std::pair<IndexT, IndexT>, std::size_t > occurences;
```

Comment:
Could be nice to have a comment on the content:
// occurences: {(3dPoint, 2dPoint): count}
Or
// occurences: {(2dPoint, 3dPoint): count}